ffi/android/README.md

```diff
@@ -1,33 +1,38 @@
-# Building the Android Bindings
+# Android bindings
 
-* Install bash
-* Install rustup
-* Install Swig 4.0
-* Install cmake (used to build zmq)
+## Build
+
+In order to build Android bindings, from the project root follow the _Local_ or
+_In docker_ instructions.
```

Comment:
Do you know that you can reference MD headers by doing stuff like `[Local](#local)`

Reply:
Sure, I know it, I just didn't think about it, sorry, good hint ;) I will do it also on node js doc (when refactoring documentation of the whole project)